daml-script/dump/src/main/scala/com/daml/script/dump/TreeUtils.scala

```diff
@@ -15,6 +17,21 @@ import scalaz.syntax.foldable._
 object TreeUtils {
   final case class Selector(i: Int)
 
+  /** Sort the active contract set topologically,
+    *  such that a contract at a given position in the list
+    *  is only referenced by other contracts later in the list.
+    */
+  def topoSortAcs(acs: Map[String, CreatedEvent]): List[CreatedEvent] = {
+    val graph: Graphs.Graph[String] = acs.mapValues(createdReferencedCids)
```

Comment:
```suggestion
    val graph: Graphs.Graph[String] = acs.view.mapValues(createdReferencedCids).toMap
```
`mapValues` is lazy and recomputes the function everytime. On Scala 2.13, this should also fail to compile (with a deprecation warning).

Reply:
Yes, CI caught it. Fixed.